control_file_gen.pl

```diff
@@ -70,7 +70,7 @@
 use Date::Calc;
 use Cwd;
 #
-
+my $nscreen=1;    # frequency of time step output to STDOUT(+) or adcirc.log(-)
```

Comment:
should this match the default, `-1000` ?

Reply:
I did it this way so that we would know immediately if there was an issue where the `NSCREEN` was not populating correctly from the default value in `io_defaults.sh`. I'll change it so the defaults match.

Reply:
Code for `control_file_gen.pl` in the PR now reflects consistent default value of -1000.